book/src/raw-block.md

```diff
@@ -2,7 +2,7 @@
 
 > ## *This page is still under active development.*
 
-**Status:** Alpha
+**Status:** Beta
```

Comment:
Also note which K8s version it's beta in.  And that it also requires the nodes to be on 1.14 too